app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { css } from '@emotion/react';
+import ArrowDownIcon from 'icons/ArrowDown';
+import { ButtonPaginationProps } from '../../types';
+import { NEXT_TYPE_BUTTON, PREV_TYPE_BUTTON } from './constants';
+
+const ButtonPaginationDesktop: React.FC<ButtonPaginationProps> = ({
+  typeButton,
+  onClickPaginationArticle,
+  title,
+}): JSX.Element => {
+  const { t } = useTranslation();
+  const isNextButton = typeButton === NEXT_TYPE_BUTTON;
+  return (
+    <button
+      css={{
+        width: '350px',
+        height: '65px',
+        border: '1px solid rgba(53, 74, 137, 0.15);',
+        ':hover': {
+          background: 'rgba(53, 74, 137, 0.05)',
+        },
+      }}
+      className="ml12 mr12 mb12"
```

Comment:
Using two css libraries at the same time is bad, rewrite it using only emotion.
Also try to move styles to separate file or object

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1385/commits/a7097e81a0e7d73ac85d3d914cf0d34be5df315d

Reply:
Confirm